devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -298,6 +302,7 @@ def refresh() {
 			zigbee.readAttribute(THERMOSTAT_CLUSTER, THERMOSTAT_RUNNING_STATE) +
 			zigbee.readAttribute(FAN_CONTROL_CLUSTER, FAN_MODE) +
 			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_VOLTAGE) +
+			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, BATTERY_PERCENTAGE_REMAINING) +
```

Comment:
It might be a good idea, at least for now, to protect this with an `if danfoss` statement. There is a chance that a device (either now or in the not-so-distant future) using this DTH will support reporting both and we might end up getting two battery events that differ (i.e. it reports a percentage and reports a voltage and our percentage calc doesn't completely agree with their calc).

Reply:
Good point, i'll fix that.